core/arch/riscv64/emit_utils.c

```diff
@@ -33,13 +33,8 @@
 #include "../globals.h"
 #include "arch.h"
 
-#ifdef RISCV_ISA_C
-#    define RAW_NOP_INST 0x0001
-#    define RAW_NOP_INST_SZ 2
-#else
-#    define RAW_NOP_INST 0x00000013
```

Comment:
Do we need to keep this define (with a different name) and have DR use it when it dynamically identifies that C is not available?

Reply:
It would be safer to do so, yes. I can't find a proof in the spec that compressed instructions should be treated as NOPs on a non-C platform. I'll re-add the define and add an appropriate comment in `fill_with_nops()`.

For context, chapter "26.3 Extensions within fixed-width 32-bit instruction format" only mentions that the Compressed instruction space is available for custom extensions if C-extension is not implemented on a given platform.